### ISSUE TITLE
Add a way to disable autoscroll

### ISSUE THREAD

**Issue by ViRb3** [sindresorhus/caprine#1388]
We need a way to disable this feature - it is impossible to read history in an active conversation, as it will force your view to the bottom on every new message.

**Comment by dusansimic** [comment]
@sindresorhus I agree. I could make an option to disable auto-scroll and add a key-binding so it's faster to enable/disable it.

**Comment by sindresorhus** [comment]
Why an option though? It seems like good default behavior to only auto-scroll if the scroll position is already at the bottom. Imagine if your email app suddenly scrolled to the top every time you got a new email, even though you had it scrolled far down looking for something.

**Comment by dusansimic** [comment]
That's a good point. I could write a PR to implement that.

**Comment by dusansimic** [comment]
It seems like messenger.com already has that behavior. When a message is received and user is at the bottom of the chat, chat scrolls into view of the last received message. when user has scrolled up so last message can't be seen, on new message the chat doesn't scroll and stays where it is. As I understand this is the requested behavior by this issue. The autoscroll fix was actually adding autoscroll on new message even if user has scrolled up. I do experience an occasional bug that chat doesn't scroll when I open a new conversation so we can leave that in but to me it seems like the autoscroll fix actually broke the expected behavior.

**Comment by czerwinski76** [comment]
I hope this auto scroll will desepear soon in v2.49.

**Comment by AggelosM** [comment]
This is really annoying, it's impossible to read past messages on an active conversation :(

---

Caprine 2.50.0
Electron 10.1.1
win32 10.0.19041
Locale: en-US

**Comment by dusansimic** [comment]
@AggelosM sorry about the inconvenience. I know that this issue has been open for quite some time, it's been bugging me too. Right now I can only imagine under how much pressure and stress @sindresorhus  is because of Hacktoberfest so I'd really be glad if we could give him a bit of time to handle everything else he's working on which is a lot of stuff. When this wave of pull requests is over I hope Sindre will have more free time for this so we could close this issue.
